tools/flowc/flowc_lsp.flow

```diff
@@ -254,7 +254,8 @@ fcLspTextDocumentDocumentSymbol(call: FcLspMethodCall) -> void {
 					out = fcOutput(call.conf.threadId);
 					clearFcOutput(call.conf.threadId);
 					if (fcDetectLspError(out, call.conf)) {
-						call.out_cb(fcLspErrorResponse(call.id, 1, out));
+						//call.out_cb(fcLspErrorResponse(call.id, 1, out));
```

Comment:
Remove instead of uncommenting. Multiple times

Reply:
done